modules/png-extraction/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ravirahar/arch-python-gdcm
+ARG DICOMHome="/opt/data/new-study"
```

Comment:
This should take from a config.json value that is provided by the user, copied from the host to the container. Not a hard-coded value.

Reply:
It is just the default value and not hard coded value. User is not to touch Dockerfile instead use png-extraction-docker script which has options to set both DICOMHome and OutputDirectory.